ext/imap/php_imap.c

```diff
@@ -830,6 +835,17 @@ PHP_FUNCTION(imap_reopen)
 		RETURN_THROWS();
 	}
 
+	if (options && ((options & ~(OP_READONLY | OP_ANONYMOUS | OP_HALFOPEN | OP_EXPUNGE | CL_EXPUNGE)) != 0)) {
```

Comment:
How do you determine which options are valid for which function?

Reply:
I pulled them from the PHP.net documentation, as I would expect the options which are documented there to work.

Reply:
The question then is whether imap_reopen only lists part of the flags because the rest are not supported, or because only the imap_open docs were updated ^^